tox.ini

```diff
@@ -23,3 +23,15 @@ basepython = python3.7
 
 [testenv:py38]
 basepython = python3.8
+
+
+[testenv:checks]
+basepython = python3.8
+deps =
+    black>=19.10b0
+    flake8>=3.7.9
+    mypy>=0.770
+commands =
+    flake8 --exclude=.git,.tox,venv* itemadapter tests
+    mypy --ignore-missing-imports --follow-imports=skip itemadapter
+    black --check itemadapter tests
```

Comment:
Wouldn’t it be better to keep these 3 commands in 3 separate Tox environments?

Otherwise, I believe the first failing one will prevent the rest from running, potentially hiding additional issues.